src/main/java/net/dv8tion/jda/api/requests/restaction/TestEntitlementCreateAction.java

```diff
@@ -0,0 +1,86 @@
+package net.dv8tion.jda.api.requests.restaction;
+
+import net.dv8tion.jda.api.entities.Entitlement;
+import net.dv8tion.jda.api.requests.RestAction;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Extension of {@link net.dv8tion.jda.api.requests.RestAction RestAction} specifically
+ * designed to create a {@link Entitlement Entitlement}.
+ * This extension allows setting properties before executing the action.
+ *
+ * @see    net.dv8tion.jda.api.JDA
+ * @see    net.dv8tion.jda.api.JDA#createTestEntitlement(long, long, OwnerType)
+ */
+public interface TestEntitlementCreateAction extends RestAction<Entitlement>
+{
+
+    /**
+     * Set the SKU's id to create the entitlement in
+     *
+     * @param skuId
+     *        The id of the SKU
+     */
+    void setSkuId(long skuId);
```

Comment:
We *always* return the same instance for chaining like this:

```java
createTestEntitlement()
  .setSkuId(...)
  .queue();
```

This does not work when you return void.